sw/device/lib/testing/edn_testutils.c

```diff
@@ -24,9 +25,21 @@ dif_edn_seed_material_t edn_testutils_seed_material_build(void) {
 /**
  * Returns a randomized EDN auto mode configuration.
  */
-dif_edn_auto_params_t edn_testutils_auto_params_build(void) {
-  dif_edn_seed_material_t seed0 = edn_testutils_seed_material_build();
-  dif_edn_seed_material_t seed1 = edn_testutils_seed_material_build();
+dif_edn_auto_params_t edn_testutils_auto_params_build(bool disable_rand) {
+  dif_edn_seed_material_t seed0 =
+      edn_testutils_seed_material_build(disable_rand);
+  dif_edn_seed_material_t seed1 =
+      edn_testutils_seed_material_build(disable_rand);
+  dif_edn_seed_material_t seed2 =
+      edn_testutils_seed_material_build(disable_rand);
+  unsigned int glen = disable_rand ? 1
+                                   : rand_testutils_gen32_range(
+                                         0, kDifEntropySeedMaterialMaxGlen);
+  unsigned int num_reqs_between_reseeds =
+      disable_rand
+          ? 4
+          : rand_testutils_gen32_range(0, kDifMaxNumReqsBetweenReseeds);
```

Comment:
I think we should take a lower maximum value here. `kDifMaxNumReqsBetweenReseeds` is very high (0xffff_ffff). I am not sure if any of our tests run more than e.g. 128 reqs.

Reply:
Thanks, I agree, I'll set it to a lower number in the next push.